src/entities/user/lib/authorization.service.ts

```diff
@@ -37,6 +44,21 @@ export class AuthorizationService {
 
     return this.httpService.PUT(`/user/password/temporary?${query}`)
   }
+
+  public checkTemporaryPassword(data: ICheckTemporaryPasswordPayload) {
+    const params = {
+      token: data.temporaryToken,
+    }
+
+    return this.httpService.GET(`/user/password/temporary/${data.userId}`, { params })
+  }
+
+  public updatePassword(data: IUpdatePasswordPayload) {
+    const headers = { "Girder-Token": data.token }
+    const params = { old: data.oldPassword, new: data.newPassword }
```

Comment:
Please make sure that IUpdatePasswordPayload accepts { old, new } so that you don't have to make transformations inside the method.

Reply:
Done